### DISCUSSION TITLE
if it is support torch.nn.parallel.DistributedDataParallel oneday

### DISCUSSION THREAD

**Discussion by STsingerST** [pyg-team/pytorch_geometric#6156] [General]
Now I am use _torch_geometric.nn.DataParallel_ which is from _torch.nn.DataParallel_, but it is very slow even if use many GPUs.
If it is possible to support _torch.nn.parallel.DistributedDataParallel_ oneday in PyG?

**Comment by LukeLIN-web** [comment]
See   [pytorch_geometric/blob/master/examples/multi_gpu/distributed_sampling.py](https://github.com/pyg-team/pytorch_geometric/tree/master/examples/multi_gpu)
